### PR TITLE
Avoid cumbersome formatter workaround

### PR DESCRIPTION
**Pull request by lrhn**
If a list shouldn't be formatted with one element on each line, a single EOL-comment inside is enough to disable the formatter. Much simpler.

### CODE REVIEW

File: packages/flutter_markdown/test/image_test_mocks.dart [hunk 1]
```diff
@@ -69,25 +69,15 @@ MockHttpClient createMockImageHttpClient(SecurityContext? _) {
...
-  ''';
-
-// Convert the string representing the hexidecimal bytes in the image into
-// a list of integers that can be consumed as image data in a stream.
-final List<int> _transparentImage = const LineSplitter()
-    .convert(_imageBytesAsString.replaceAllMapped(
-        RegExp(r' *0x([A-F0-9]{2}),? *\n? *'), (Match m) => '${m[1]}\n'))
-    .map<int>((String b) => int.parse(b, radix: 16))
-    .toList();
+];
```
Comment (lrhn) [inline packages/flutter_markdown/test/image_test_mocks.dart:80]: Another, still simpler, workaround would be:
```dart
final String _imageBytesString =
  '\x89\x50\x4E\x47\x0D\x0A\x1A\x0A\x00\x00\x00\x0D\x49'
  '\x48\x44\x52\x00\x00\x00\x01\x00\x00\x00\x01\x08\x06'
  '\x00\x00\x00\x1F\x15\xC4\x89\x00\x00\x00\x0A\x49\x44'
  '\x41\x54\x78\x9C\x63\x00\x01\x00\x00\x05\x00\x01\x0D'
  '\x0A\x2D\xB4\x00\x00\x00\x00\x49\x45\x4E\x44\xAE';
final List<int> _transparentImage = [..._imageBytesString.codeUnits];
```

Having a full, formatted string, and parsing it using RegExp, just to get to some bytes, feels a little like overkill.

**Comment by stuartmorgan** [comment]
@lrhn Are you planning on updating this to pass the analyzer CI check?

**Comment by lrhn** [comment]
Merged to head, removed unused import.

**Comment by stuartmorgan** [comment]
You're still missing a type specifier on the list, so it fails analysis with Flutter settings.

**Comment by lrhn** [comment]
Added type to list literal. Now I'm half-expecting to get a warning about having types on both sides of the `=`. Let's see what Mr. Analyzer says.

**Review by stuartmorgan — APPROVED** [review]
LGTM

**Comment by stuartmorgan** [comment]
@lrhn You might want to request being added to flutter-hackers. Meanwhile, @domesticmouse could you approve this?

**Review by tarrinneal — APPROVED** [review]

**Review by bparrishMines — APPROVED** [review]
LGTM

**Pull request merged**